docs/_assets/style.css

```diff
@@ -12,12 +12,37 @@ mdjs-preview {
   margin-bottom: 20px;
 }
 
-#main-header a[href='/blog/'] {
-  display: none;
+#main-header a {
+  font-weight: normal;
 }
 
-@media screen and (min-width: 1024px) {
+#main-header a.current.active {
+  font-weight: bold;
+}
+
+#main-header .current.active {
+  display: block;
+}
+
+@media only screen and (max-width: 600px) {
+  #main-header a {
+    font-size: 12px;
```

Comment:
By making the anchors this small the click-area will not be big enough.

I guess you tried to keep all elements in the header to be sure every page is still reachable, but the blog and our github are also reachable via the footer. Preferably they are also available under the hamburger menu.

Reply:
Yes, I tried to keep all anchors on the header for consistency across devices/viewports.
"but the blog and our github are also reachable via the footer"
In that case, is it better to remove the blog and github anchors from the header altogether?

Reply:
If i remove the github anchor and change the padding left and right of the header, i was able to increase the font-size to 14px. Which looks acceptable:

![image](https://user-images.githubusercontent.com/10164421/114668769-6762fa00-9d01-11eb-8a54-5f262493dd50.png)

If you play around with the padding you can get click areas of 44px or bigger.

Reply:
Then it's still not consistent among all viewports (4 links VS 5), which defeats the purpose. so I'd rather leave it as is fot the smaller viewports, since the blog & github are reachable via hamburger menu and footer.
Alternative: remove blog & github links from header altogether - in order to achieve consistency.